tests/page-objects/contacts/contacts.wdio.page.js

```diff
@@ -286,5 +344,8 @@ module.exports = {
   createNewAction,
   openReport,
   getContactCardTitle,
-  getContactInfoName
+  getContactInfoName,
+  getContactMedicID,
+  addHealthPrograms,
+  getImmCardVaccinesValues
```

Comment:
```suggestion
  getImmCardVaccinesValues,
```
Next time someones add a new function, git will show 1 change in the diff.